common.h

```diff
@@ -253,8 +253,10 @@ typedef unsigned long BLASULONG;
 
 #ifdef USE64BITINT
 typedef BLASLONG blasint;
+#define blasabs(x) labs(x)
```

Comment:
This doesn't seem like it will work on Windows, which is an ILP64 architecture where `long == int` and you have to use `llabs`.